internal/config/config.go

```diff
@@ -133,6 +135,7 @@ func loadConfigFromFlags(args []string) (result Config, configFile string, err e
 	flags.StringVarP(&result.ServerURL, "server", "s", "", "URL to Nextcloud server.")
 	flags.StringVarP(&result.Username, "username", "u", defaults.Username, "Username for connecting to Nextcloud.")
 	flags.StringVarP(&result.Password, "password", "p", defaults.Password, "Password for connecting to Nextcloud.")
+	flags.BoolVar(&result.TLSNoVerify, "tls-no-verify", defaults.TLSNoVerify, "Skip certificate verification of Nextcloud server.")
```

Comment:
Should we call the option `--tls-skip-verify` instead? Then it would just be the same as the actual option on the HTTP client. Same with the environment variable and YAML configuration.

Reply:
Fine for me.